### PR TITLE
build: Upgrade flutter version

### LINKED ISSUE #268: Upgrade flutter version
As we work on implementing design Vlad noticed that image color is not applied properly for icons but that seems to be fixed on the master branch - see https://github.com/flutter/flutter/issues/33316#issuecomment-647094397

Currently, we want to upgrade to the latest beta channel version.

Currently, we'll put proper color code inside the SVG but that's not well maintainable - so we'll need to upgrade to the new flutter version. As we didn't update for a while and flutter team changed a lot in their driver testing implementation - that might take some time (2-4 days). Here are the potential areas we'll need to change:

- [ ] Update driver tests script 
- [ ] Update the GitHub actions 
- [ ] Update the deployment doc
- [ ] Test everything is fine on the new version 
- [x] Flutter rendering issue on project groups: 1) use keyboard to jump to the checkboxes and then press up down to scroll the checkbox list, notice grey selection area moving above other elements. See screenshot: 
image

### PR DESCRIPTION
**Pull request by solid-yuriiuvarov**
Fixes #268

## Checklist

- [x] Documented
- [x] Unit tested
- [x] Component tested

### CODE REVIEW

File: metrics/web/lib/base/presentation/graphs/circle_percentage.dart [hunk 1]
```diff
@@ -94,7 +94,6 @@ class _CirclePercentageState extends State<CirclePercentage>
         final initialPadding = _getChildPadding(constraints);
 
         return Align(
-          alignment: Alignment.center,
```
Comment (solid-dimakoniaiev) [inline metrics/web/lib/base/presentation/graphs/circle_percentage.dart:97]: Maybe we can just use a Center widget for this?

File: metrics/web/test/project_groups/presentation/widgets/add_project_group_card_test.dart [hunk 2]
```diff
@@ -241,20 +241,40 @@ void main() {
...
 
-        final networkImage = FinderUtil.findNetworkImageWidget(tester);
+        final image = tester.widget<Image>(find.byType(Image));
+
+        expect(image.color, equals(inactiveIconColor));
+      },
+    );
+
+    testWidgets(
+      "applies the positive icon color form the metrics theme to the network image when the add project group card is enabled",
```
Comment (solid-vladlaba) [inline metrics/web/test/project_groups/presentation/widgets/add_project_group_card_test.dart:263]:
```suggestion
      "applies the positive icon color form the metrics theme to the add icon when the add project group card is enabled",
```

File: metrics/web/test/project_groups/presentation/widgets/add_project_group_card_test.dart [hunk 3]
```diff
@@ -241,20 +241,40 @@ void main() {
     );
 
     testWidgets(
-      "displays the disabled add icon svg when the card is disabled",
+      "applies the inactive icon color form the metrics theme to the network image when the add project group card is disabled",
```
Comment (solid-vladlaba) [inline metrics/web/test/project_groups/presentation/widgets/add_project_group_card_test.dart:244]:
```suggestion
      "applies the inactive icon color form the metrics theme to the add icon when the add project group card is disabled",
```

**Review by solid-vladlaba — CHANGES_REQUESTED** [review]
Tremendous work guys, @solid-yuriiuvarov @solid-dimakoniaiev! 👍 
Like changes related to a new version and lint related changes as well. I have a couple of suggestions below - let me know what you think!

File: metrics/web/lib/common/presentation/user_menu_button/theme/user_menu_button_theme_data.dart [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+/// A class that stores the theme data for the user menu button.
+class UserMenuButtonThemeData {
+  /// A [Color] to use when the [MetricsUserMenu] is open.
+  final Color activeColor;
+
+  /// A [Color] to use when the [MetricsUserMenu] is close.
+  final Color inactiveColor;
+
+  /// Creates the [UserMenuButtonThemeData].
+  const UserMenuButtonThemeData({this.activeColor, this.inactiveColor});
```
[the remark this answers is not in the document]
Comment (solid-dimakoniaiev) [reply metrics/web/lib/common/presentation/user_menu_button/theme/user_menu_button_theme_data.dart:12]: Yes, we're sure)

**Review by solid-vladlaba — APPROVED** [review]
LGTM, thanks @solid-dimakoniaiev @solid-yuriiuvarov!:rocket: 
Perfect work! Love changes with TappableArea 🙂

**Pull request merged**